network-api/networkapi/templates/fragments/buyersguide/featured_page.html

```diff
@@ -1,5 +1,5 @@
 {% load static wagtailcore_tags wagtailimages_tags i18n %}
-<div id="featured-article" class="tw-row">
+<div id="featured-pages" class="tw-row">
```

Comment:
@tbrlpld  In response to your comment regarding the ID change from `featured-article` to `featured-pages`:

After speaking with design, I found that the ID is used for GTM to track link clicks in this DIV. 

I have the OK from design to change this to `featured-pages` as long as I update GTM to match when this gets merged.

Thanks!

Reply:
Thanks for checking @danielfmiranda. Since you already need to change it in GTM to, I would suggest to also move to the more obvious `data-gtm` approach (which makes clear why it exists). We might want to add a the `data-gtm` approach to the docs somewhere 🤔

Reply:
Sounds good! I have updated it to now use `data-gtm` instead of `id`. 

Regarding updating the docs, I will create a ticket to update them!